systemd/tinc@.service.in

```diff
@@ -4,6 +4,9 @@ Documentation=info:tinc
 Documentation=man:tinc(8) man:tinc.conf(5)
 Documentation=http://tinc-vpn.org/docs/
 PartOf=tinc.service
+Before=systemd-user-sessions.service
+After=network-online.target
```

Comment:
I am strongly against depending on `network-online.target` by default as this target might be never reached if either networkd or network-manager do not get a DHCP address. Tinc would be still useful if there are other nodes in the same ethernet segment, which it can connect to through broadcast discovery.
If someone wants to create devices before starting tinc a better way is to add dependencies on devices units.
Tinc is decentral and does not depend on a default gateway hence `network-online.target` is not needed.